v63005/outref/gtm5574.txt

```diff
@@ -0,0 +1,64 @@
+# Conversions using a random 18 digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 19 digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 20 digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 255 digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 256 digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+##TEST_AWK# Conversions using a random [0-9]* digit number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 5 digit negative number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
+# Conversions using a random 10 digit negative number
+%DH-----Correct Conversion
+%HO-----Correct Conversion
+%OD-----Correct Conversion
+%DO-----Correct Conversion
+%OH-----Correct Conversion
+%HD-----Correct Conversion
+-------------------------------------------
```

Comment:
Can we also test the bolded part in the release note.

Additionally, %DH and %DO routines now handle the conversion of negative decimal numbers properly, **even when the specified length is not long enough to represent that fully converted number**.